WordPress/Jetpack/Classes/ViewRelated/Login Error/View Models/JetpackNoSitesErrorViewModel.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+
+struct JetpackNoSitesErrorViewModel: JetpackErrorViewModel {
+    let image: UIImage? = UIImage(named: "jetpack-empty-state-illustration")
+    var description: FormattedStringProvider = FormattedStringProvider(string: Constants.description)
+    var primaryButtonTitle: String? = Constants.primaryButtonTitle
+    var secondaryButtonTitle: String? = Constants.secondaryButtonTitle
+
+    func didTapPrimaryButton(in viewController: UIViewController?) {
+        guard let url = URL(string: Constants.helpURLString) else {
+            return
+        }
+
+        let controller = WebViewControllerFactory.controller(url: url)
+        let navController = UINavigationController(rootViewController: controller)
+
+        viewController?.present(navController, animated: true)
+    }
+
+    func didTapSecondaryButton(in viewController: UIViewController?) {
+        viewController?.navigationController?.popToRootViewController(animated: true)
+    }
+
+    private struct Constants {
+        static let description = NSLocalizedString("If you already have a site, you’ll need to install the free Jetpack plugin and connect it to your WordPress.com account.",
```

Comment:
We can merge this PR as it is but I think we need a better message for this case.

I think it can say something like "there are no Jetpack sites linked to this account"

cc @osullivanchris